cogs/audio.py

```diff
@@ -1849,10 +1852,15 @@ def currently_downloading(self, server):
                     stop_times[server] = int(time.time())
 
                 if hasattr(vc, 'audio_player'):
-                    if (vc.audio_player.is_done() or len(vc.channel.voice_members) == 1):
+                    if (vc.audio_player.is_done() == 1):
                         if server not in stop_times or stop_times[server] is None:
                             log.debug("putting sid {} in stop loop".format(server.id))
                             stop_times[server] = int(time.time())
+                    elif self._noppl_disconnect == False:
```

Comment:
you defined this as a function above. this isn't how you call a function.
`if not a_boolean:` is preferred over `if a_boolean == False:`